web/src/components/wizard/preflight/LinuxPreflightCheck.tsx

```diff
@@ -54,7 +72,8 @@ const LinuxPreflightCheck: React.FC<LinuxPreflightCheckProps> = ({ onComplete })
         },
       });
       if (!response.ok) {
-        throw new Error("Failed to run preflight checks");
+        const errorData = await response.json().catch(() => ({}));
+        throw new Error(errorData.message || "Failed to run preflight checks");
```

Comment:
If I'm reading correctly, we're ignoring the `json()` error here and has a fallback returning an Error `"Failed to run preflight checks"`.

- I wonder what happens when `json()` fails, I believe errorData can be `undefined` which will result in `... errorData.message ...` to fail.
- Wondering if we want to capture the response body's `text` for analysis since this shouldn't really happen. Maybe we should bubble up that error?